### DISCUSSION TITLE
Still Executing ......................................................................

### DISCUSSION THREAD

**Discussion by Rahim-Habibi** [idaholab/moose#23799] [Q&A Modules: Solid mechanics] [answered]
Hi, 
I wrote a terminator based on "Terminator.C".  The terminator does not get value from a postprocessor, actually the value for termination is being determined inside the terminator. 
In my model, the terminator should terminate the run when the value reaches to 1. I checked the terminator in small box model which has less 100 elements and it successfully terminated as expected. Then I applied it in my large-scale reservoir model which has more 200k elements. Based on the previous runs, the termination meets the termination condition during 4th time step (value exceeds 1 in 4th time steps). So, the run should terminate after 4 time steps, here it was tracked as postprocessor named ratio. But, as you can see in figures below, after converging, it shows "Still Executing..." which seems a kind of end-less loop (I do not know what is actually calculating at this step). I left it to keep executing, but it did not finish the executing. 
Also, I have looked up "Still Executing" im MOOSE source file, but I have found any line or command. 
figures below show the terminal when I used terminator in SOFT and HARD fail modes resepectively.  

here is the terminator block in input file:
```
[./terminator]
    type = TigerTerminator
    total_stress = stress
    criterion_type = Slip_Tendency
    variable = fcn
    fail_mode = SOFT
    use_displaced_mesh = false
    execute_on = ' TIMESTEP_END'
    block = '11 12'
  [../]
```
![StillExe](https://user-images.githubusercontent.com/57484835/226567309-a74a221b-441f-43ed-9942-fd866be034b4.PNG)
![StillExe-HARD](https://user-images.githubusercontent.com/57484835/226572563-44cb225e-6dd5-4414-8881-6e953c7b2209.PNG)

**Comment by GiudGiud** [comment, marked as answer]
hello

It s probably hanging at the execution of the user objects.
Is your code open source? Can we have a look?

Guillaume
